test/specs/map/MapService.spec.js

```diff
@@ -2035,6 +2036,7 @@ describe('ga_map_service', function() {
       });
     });
 
+    /* TODO: fix this test
```

Comment:
I tried to fix this one but no luck ($timeout.flush, $rootScope.digest etc..) It works fine in fact, but the promise never resolves in the tests.